uk/lessons/basics/enum.md

```diff
@@ -0,0 +1,192 @@
+---
+version: 1.3.0
+title: Enum
+---
+
+Набір алгоритмів для операцій з перечисленнями.
+
+{% include toc.html %}
+
+## Enum
+
+Модуль `Enum` включає в себе білшь ніж 70 функцій для роботи з перечисленнями. Всі колекції, про які ми дізналися в [попередньому уроці](../collections/), за виключенням кортежів, є перечисленнями.
+
+Цей урок містить лише невелику частину доступних функцій, проте ми можемо вивчити їх самостійно.
+Давайте проведемо невеличкий експеримент в IEx.
+
+```elixir
+iex> Enum.__info__(:functions) |> Enum.each(fn({function, arity}) ->
+...>   IO.puts "#{function}/#{arity}"
+...> end)
+all?/1
+all?/2
+any?/1
+any?/2
+at/2
+at/3
+...
+```
+
+Стає зрозуміло, що у нас є велика кількість функцій.
```

Comment:
Why do you use separate lines here?

Reply:
Because same formatting i saw in other translations. In english too. Just left the same structure.

Reply:
Ok, got it.